cmd/generator/main.go

```diff
@@ -16,8 +16,65 @@ limitations under the License.
 
 package main
 
-import "fmt"
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/iancoleman/strcase"
+
+	"github.com/crossplane-contrib/terrajet/pkg/pipeline"
+	"github.com/pkg/errors"
+	"github.com/terraform-providers/terraform-provider-aws/aws"
+)
+
+const (
+	ModulePath  = "github.com/crossplane-contrib/provider-tf-aws"
+	GroupSuffix = ".aws.tf.crossplane.io"
+)
 
 func main() {
-	fmt.Println("Hello World!")
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(errors.Wrap(err, "cannot get working directory"))
+	}
+	groups := map[string]map[string]*schema.Resource{}
+	for name, resource := range aws.Provider().ResourcesMap {
+		if len(resource.Schema) == 0 {
+			// There are resources with no schema, like aws_securityhub_account
+			// that we will address later.
```

Comment:
nit: 
It might be helpful to add some log messages in general like `generating resource x` or specifically here like `skipping resource y since it has no schema`

Reply:
Great idea!

Reply:
`generating resource x` makes the log too long to catch the skipped ones. I opted for printing the total count.